src/main/resources/layouts/manage-project.fxml

```diff
@@ -22,26 +23,29 @@
       <RowConstraints vgrow="SOMETIMES" />
       <RowConstraints vgrow="SOMETIMES" />
       <RowConstraints vgrow="SOMETIMES" />
+      <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
       <RowConstraints vgrow="SOMETIMES" />
       <RowConstraints vgrow="SOMETIMES" />
       <RowConstraints vgrow="SOMETIMES" />
   </rowConstraints>
    <children>
       <Label text="Name:" />
       <Label text="Color:" GridPane.rowIndex="3" />
-      <Label text="IsWork:" GridPane.rowIndex="4" />
-      <Label text="SortIndex:" GridPane.rowIndex="5" />
+      <Label text="IsWork:" GridPane.rowIndex="5" />
+      <Label text="SortIndex:" GridPane.rowIndex="6" />
       <TextField fx:id="nameTextField" GridPane.columnIndex="1" />
       <ColorPicker fx:id="textFillColorPicker" GridPane.columnIndex="1" GridPane.rowIndex="3" />
```

Comment:
can you create the "random" button next to the color?
Like:
![image](https://user-images.githubusercontent.com/5772511/204241129-527a0e01-0cae-4451-adf0-79012356453d.png)
(maybe such a "random" con exists in font-awesome icons :))